src/server/templates/end.njk

```diff
@@ -2,7 +2,20 @@
 
 {% block body_main_content %}
   <div class="page-section">
-    <h1>Thank you</h1>
-    <p>Goodbye</p>
+    <h1 class="heading-medium">Thank you for applying to register</h1>
+
+    <div class="reading-width">
+      <p>They are dealing with your registration application. They will contact
```

Comment:
Yes, this will be done in #41. Wanted to get this one in first and will be adding all the replacements in that PR.